public/styles/events.css

```diff
@@ -0,0 +1,225 @@
+.event_header {
+    opacity: 1;
+}
+
+.event_title{
+    text-align: left;
+    font: Bold 50px Montserrat;
```

Comment:
I see you used the "Montserrat" font family a lot. There's this [issue](https://github.com/she-code-africa/SCA-Website-V2/issues/14) for making it the website's main font. Can you swap this out for us at once so you don't get to repeat it a lot?

Reply:
Also, there is this [variables](https://github.com/she-code-africa/SCA-Website-V2/blob/develop/public/styles/variables.css) file for common styles. You'll see some color codes you used (change it to use the variables instead. You can also add other color codes so we can limit repetition as much as possible

Reply:
Sub texts are now bigger, additional white spaces have been added between sub sections, variable names for the color codes used, and the app's default font face is now Montserrat as in Issue #14

Reply:
![Screenshot from 2020-08-05 10-34-22](https://user-images.githubusercontent.com/31415689/89397101-5213e580-d707-11ea-812d-6e1a9055768a.png)